storage/src/vespa/storage/common/statusmetricconsumer.cpp

```diff
@@ -155,160 +75,76 @@ StatusMetricConsumer::reportStatus(std::ostream& out,
             updateSnapshotHooks ? ", calling snapshot hooks too" : ".");
         _manager.updateMetrics(updateSnapshotHooks);
     } else {
-        LOG(debug, "Not calling update hooks as dontcallupdatehooks option "
-                   "has been given");
+        LOG(debug, "Not calling update hooks as dontcallupdatehooks option has been given");
     }
     framework::SecondTime currentTime(_component.getClock().getTimeInSeconds());
-    bool html = (!path.hasAttribute("format")
-                 || path.getAttribute("format") == "html");
-    bool xml = (!html && path.getAttribute("format") == "xml");
-    bool json = (!html && path.getAttribute("format") == "json");
+    bool xml = (path.getAttribute("format") == "xml");
+    bool json = (path.getAttribute("format") == "json");
 
     int verbosity(path.get("verbosity", 0));
         // We have to copy unset values if using HTML as HTML version gathers
         // metrics for calculations and thus needs unset values.
-    bool copyUnset = (html || verbosity >= 2);
+    bool copyUnset = (verbosity >= 2);
```

Comment:
I wonder about this verbosity >= 2 which triggers copying and constitutes 2/3 of the remaining complexity.
Is that necessary or could it be removed too ? Less is more.

Reply:
K, I will leave this as is for now.